### PR TITLE
Add storage config for cloud

### PR DESCRIPTION
**Pull request by johnosullivan**
# Pull Request

## Description

Fixes bug of replica deployments with a single schema store on volumes. 

### Basic

- [x] Did you write tests for the code in this PR?
- [x] Did you document your changes in the README and/or in docstrings (as needed)?

### Security

- [ ] Authorization has been implemented across these changes
- [ ] Injection has been prevented (parameterized queries, no eval or system calls)
- [ ] Any web UI is escaping output (to prevent XSS)
- [ ] Sensitive data has been identified and is being protected properly

## Notes for the Reviewer

N/A

### CODE REVIEW

File: data_resource/admin/app.py [hunk 1]
```diff
@@ -82,15 +85,15 @@ def shutdown_session(exception=None):
         return app
 
 
+# TODO REFACTOR
```
Comment (loganripplinger) [inline data_resource/admin/app.py:88]: :D

File: data_resource/storage/aws_s3.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        self.region_name = self.config.AWS_S3_REGION
+
+        if (
+            self.aws_access_key_id == None
+            or self.aws_secret_access_key == None
+            or self.region_name == None
+        ):
+            raise RuntimeError("Invalid AWS S3 Configuration")
+
+    def required_env_check(self):
```
Comment (loganripplinger) [inline data_resource/storage/aws_s3.py:24]: If these or others are private methods prepend them with an underscore,

`def required_env_check(self):`
to
`def _required_env_check(self):`

File: data_resource/admin/app.py (hunk 1 above)
Comment (johnosullivan) [reply data_resource/admin/app.py]: Remove. 👍

File: data_resource/storage/aws_s3.py (hunk 2 above)
Comment (johnosullivan) [reply data_resource/storage/aws_s3.py]: Made private, might need to be public in future.

**Pull request merged**